.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "rrweb"]
 	path = rrweb
-	url = git@github.com:highlight-run/rrweb.git
+	url = https://github.com/highlight/rrweb.git
```

Comment:
```suggestion
	url = git@github.com:highlight/rrweb.git
```

Reply:
the `ssh` reference was necessary previously when our rrweb fork was private. now that it is public, we should be able to clone over https. this avoids problems for folks who do not have an ssh key configured.